Composer/packages/client/src/pages/setting/dialog-settings/index.tsx

```diff
@@ -57,9 +56,7 @@ export const DialogSettings = () => {
   };
 
   const handleChange = debounce(result => {
```

Comment:
I just noticed this but on every render, `handleChange` will be a new debounced function. It needs to be memoized.

Reply:
Wrap it in useMemo.